_includes/specials/taproot/00-bech32m.md

```diff
@@ -0,0 +1,35 @@
+Starting at block {{site.trb}}, expected in November, Bitcoin users will be
+able to safely receive payments to taproot  addresses.  Given the
+user enthusiasm for taproot and the five months that wallet
+developers have to implement support for it, Optech expects there to be
+several popular wallets that will allow their users to generate taproot
+addresses at the earliest possible moment.
+
+That means any other wallet or service that sends bitcoins to
+user-provided addresses needs to be able to send to taprot addresses by
+block {{site.trb}} or risk confusing and disappointing its users.  Pay
+to TapRoot (P2TR) addresses use [bech32m][topic bech32] as specified in [BIP350][], which
+is slightly different than [BIP173][]'s bech32 algorithm used for segwit
+v0 P2WPKH and P2WSH addresses.  The slight difference is that the
+checksum function uses a different constant, with bech32m using
+`0x2bc830a3` instead of bech32's `0x01`.
```

Comment:
"different, difference, different" in three lines. Perhaps:

```suggestion
v0 P2WPKH and P2WSH addresses.  Bech32m uses the constant
`0x2bc830a3` instead of bech32's `0x01` in the checksum function.
```